src/perf_tool_backend.ml

```diff
@@ -12,9 +12,22 @@ module Record_opts = struct
 
   let param =
     let%map_open.Command multi_thread =
-      flag "-multi-thread" no_arg ~doc:"record multiple threads"
+      flag
+        "-multi-thread"
+        no_arg
+        ~doc:
+          "Records every thread of an executable, instead of only the thread whose TID \
+           is equal to the process' PID.\n\
+           Warning: this flag decreases the trace's lookback period because the kernel \
+           divides snapshot buffer resources equally across all threads."
```

Comment:
I know I'm the one that said this, but I'm only 70% on this specifically. I know the resources are divided, just not exactly how.